packages/next/src/client/components/react-dev-overlay/internal/helpers/stack-frame.ts

```diff
@@ -102,30 +102,35 @@ export function getOriginalStackFrames(
   )
 }
 
+const webpackRegExes = [
+  /^webpack-internal:\/\/\/(\.)?(\((\w+)\))?/,
+  /^(webpack:\/\/\/(\.)?|webpack:\/\/(_N_E\/)?)(\((\w+)\))?/,
+]
+
+function isInternal(file: string) {
+  return webpackRegExes.some((regEx) => regEx.test(file))
+}
+
 /**
  * Format the webpack internal id to original file path
  * webpack-internal:///./src/hello.tsx => ./src/hello.tsx
  * webpack://_N_E/./src/hello.tsx => ./src/hello.tsx
  * webpack://./src/hello.tsx => ./src/hello.tsx
  * webpack:///./src/hello.tsx => ./src/hello.tsx
- *
  */
 function formatFrameSourceFile(file: string) {
-  return file
-    .replace(/^webpack-internal:\/\/\/(\.)?(\((\w+)\))?/, '')
-    .replace(/^(webpack:\/\/\/(\.)?|webpack:\/\/(_N_E\/)?)(\((\w+)\))?/, '')
+  return file.replace(webpackRegExes[0], '').replace(webpackRegExes[1], '')
```

Comment:
If we add new regex or adjust the length later we dont need to change this part
```suggestion
  for (const regex of webpackRegExes) file = file.replace(regex, '')
  return file
```